firebase-perf/src/main/java/com/google/firebase/perf/metrics/validator/FirebasePerfTraceValidator.java

```diff
@@ -104,6 +109,17 @@ private boolean areCountersValid(@Nullable TraceMetric trace, int deep) {
     return true;
   }
 
+  private boolean isValidScreenTrace(@NonNull TraceMetric trace) {
+    if (!trace.getName().startsWith(Constants.SCREEN_TRACE_PREFIX)) {
```

Comment:
I would move the prefix check to the caller because if a screen trace does not start with `SCREEN_TRACE_PREFIX` does not mean it is a valid screen trace.

Reply:
done 👍